CONTRIBUTING-de.md

```diff
@@ -145,14 +145,14 @@ Unsere Listen enthalten einen minimalen Satz an Metadaten: Titel, URLs, Autoren,
 - Wir wollen alle Urheber kostenloser Inhalte angemessen nennen, inklusive eventueller Übersetzer!
 - For übersetzte Werke sollte der Autor des ursprünglichen Werks genannt werden.
 - Wir erlauben keine Links für Urheber.
-- Für Sammlungen oder neu zusammengestellte Werke, benötigt der "Urheber" eventuell eine Beschreibung. Bücher von "GoalKicker" werden z.B. als "Zusammengestellt aus StackOverflow Dokumentationen" gekennzeichnet.
+- Für Sammlungen oder neu zusammengestellte Werke, benötigt der "Urheber" eventuell eine Beschreibung. Bücher von "GoalKicker" werden z. B. als "Zusammengestellt aus StackOverflow Dokumentationen" gekennzeichnet.
```

Comment:
I would not add a space between z. and B. Besides that being unusual, markdown (like HTML) flows and a line break might occur at any position. That means, you could end up with `z.` on the end of the line and `B.` on the start of the next line, which might be confusing for the reader.

Reply:
You have an argument with the line break example, but still it would be incorrect.

See [DIN-5008](https://www.din-5008-richtlinien.de/startseite/abkuerzungen/) and [the Duden](https://www.duden.de/rechtschreibung/z__B_).

Reply:
Consider using `z.&nbsp;B.` then.

Reply:
Makes sense :) Done :)